frontend/pages/policies/edit/components/PolicyAutomations/PolicyAutomations.tests.tsx

```
@@ -0,0 +1,169 @@
+      const onAddAutomation = jest.fn();
+      renderWithAppContext(
+        <PolicyAutomations
+          storedPolicy={createMockPatchPolicy()}
+          currentAutomatedPolicies={[]}
+          onAddAutomation={onAddAutomation}
+        />
+      );
+
+      expect(
+        screen.getByText(/Automatically patch Zoom/)
+      ).toBeInTheDocument();
```

ESLint

[error] Replace `⏎········screen.getByText(/Automatically·patch·Zoom/)⏎······` with `screen.getByText(/Automatically·patch·Zoom/)`

(prettier/prettier)